l10n_co_res_partner/models/l10n_co_res_partner.py

```diff
@@ -0,0 +1,459 @@
+import re
+
+
+class CountryStateCity(models.Model):
+    """
+    Model added to manipulate separately the cities on Partner address.
+    """
+    _description = 'Model to manipulate Cities'
+    _name = 'res.country.state.city'
+
+    code = fields.Char('City Code', size=5, help='Code DANE - 5 digits-',
+                       required=True)
+    name = fields.Char('City Name', size=64, required=True)
+    state_id = fields.Many2one('res.country.state', 'State', required=True)
+    country_id = fields.Many2one('res.country', 'Country', required=True)
+    _order = 'code'
+
+
+class PartnerInfoExtended(models.Model):
+    _name = 'res.partner'
+    _inherit = 'res.partner'
+
+    # Company Name
+    companyName = fields.Char("Name of the Company")
+
+    # companyType
+    companyType = fields.Selection(related='company_type')
+
+    # Adding new name fields
+    name1 = fields.Char("First Name")
+    name2 = fields.Char("Second Name")
+    lastname1 = fields.Char("Last Name")
+    lastname2 = fields.Char("Second Last Name")
+
+    # Document information
+    doctype = fields.Selection(
+        [
+            (1, "No identification"),
+            (11, "11 - Birth Certificate"),
+            (12, "12 - Identity Card"),
+            (13, "13 - Citizenship Card"),
+            (21, "21 - Alien Registration Card"),
+            (22, "22 - Foreigner ID"),
+            (31, "31 - TAX Number (NIT)"),
+            (41, "41 - Passport"),
+            (42, "42 - Foreign Identification Document"),
+            (43, "43 - No Foreign Identification")
+
+        ], "Type of Identification"
+    )
+    xidentification = fields.Char("Document Number", store=True,
+                                  help="Enter the Identification Number")
+    verificationDigit = fields.Integer('VD', size=2)
+    formatedNit = fields.Char(
+        string='NIT Formatted',
+        compute="_compute_concat_nit",
+        store=True
+    )
+
+    # Tributate regime
+    pn_retri = fields.Selection(
+        [
+            (6, "Simplified"),
+            (23, "Natural Person"),
+            (7, "Common"),
+            (11, "Great Taxpayer Autorretenedor"),
+            (22, "International"),
+            (25, "Common Autorretenedor"),
+            (24, "Great Contributor")
+        ], "Tax Regime"
+
+    )
+
+    # CIIU - Clasificación Internacional Industrial Uniforme
+    ciiu = fields.Many2one('ciiu', "ISIC Activity")
+    personType = fields.Selection(
+        [
+            (1, "Natural"),
+            (2, "Juridical")
+        ],
+        "Type of Person",
+        default=1
+    )
+
+    # Replacing the field company_type
+    company_type = fields.Selection(
+        [
+            ('person', 'Individual'),
+            ('company', 'Company')
+        ]
+    )
+
+    # Boolean if contact is a company or an individual
+    is_company = fields.Boolean(string=None)
+
+    # Verification digit
+    dv = fields.Integer(string=None, store=True)
```

Comment:
try to use help in ALL files and more important if they are acronymous of somthing, explaining the why and the how you will use it.